server/backend/api/app/main.py

```diff
@@ -87,7 +90,7 @@ def main():
     return RedirectResponse(url="/docs/")
 
 
-apiPrefix = '/api/v1'
+apiPrefix = ''
```

Comment:
The prefix was screwing up the path management, and version management shouldn't be hard-coded in the code IMHO. Using API-Gateway stages makes it a configuration instead of a hard-coded value.

Reply:
This might cause issues for the front-end, at least for local development, as it is expecting this prefix. But that should be an easy fix.